### PR TITLE
27 update dashboard layout

### PR DESCRIPTION
**Pull request by monazhu**
Cleaned up the dashboard layout so it's more intuitive / user-friendly and made everything fit on the same page 

image

### CODE REVIEW

File: src/app.py [hunk 1]
```diff
@@ -171,7 +177,7 @@ def update_fun_fact(year, country):
         df, country, start_year=year[0], end_year=year[1]
     )
     num_empire_state_buildings = hp.get_number_of_esb(total_co2)
-    return f"Equivalent to {num_empire_state_buildings:,} Empire State Buildings!"
+    return f"This is quivalent to {num_empire_state_buildings:,} Empire State Buildings in weight!"
```
[the remark this answers is not in the document]
Comment (monazhu) [reply src/app.py:180]: Okay thanks for the clarification!

[the remark this answers is not in the document]
Comment (farrandi) [reply src/app.py:180]: Also typo for "quivalent" -> "equivalent"

**Comment by monazhu** [comment]
@farrandi I've corrected the fun fact / typo, thanks!

let me know if there's anything else I should change in the layout!

**Review by farrandi — APPROVED** [review]
Did some minor formatting with python linter.

LGTM

**Pull request merged**